<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>689</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_newBeginRowId">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_BeginProcess">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>460</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Begin Compare</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_newSheetNames">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sheet1,Sheet2,Sheet3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_newKeyColId">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_newValueColId">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_newIfJudgeSpecified">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>是否需要根据表格内容筛选，比如只选择表格中3班的学生。如果需要就打钩</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_newJudgeOptions">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1班,2班,3班</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_newJudgeColId">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_baseSheetNames">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>120</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sheet3,Sheet4,Sheet5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_baseKeyColId">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_baseValueColId">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_baseIfJudgeSpecified">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>290</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>是否需要根据表格内容筛选，比如只选择表格中3班的学生。如果需要就打钩</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_baseJudgeOptions">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>410</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2班,3班,4班</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_baseBeginRowId">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_baseJudgeColId">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>320</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>new中需要对比的表格，如果是多个表格以英文逗号分隔</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>需要对比的起始行，从1开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>需要对比的关键字所在列，比如按照姓名-学号对比，这里填姓名所在列</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>需要对比的内容所在列，比如按照姓名-学号对比，这里填学号所在列</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>筛选内容所在列，比如选择3班的同学，就把“3班”所在列填入</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>筛选内容比如选择3班的同学，就把“3班”填入，如果是多个表格以英文逗号分隔</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>需要对比的起始行，从1开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>250</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>需要对比的内容所在列，比如按照姓名-学号对比，这里填学号所在列</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>210</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>需要对比的关键字所在列，比如按照姓名-学号对比，这里填姓名所在列</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>320</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>筛选内容所在列，比如选择3班的同学，就把“3班”所在列填入</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>390</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>筛选内容比如选择3班的同学，就把“3班”填入，如果是多个表格以英文逗号分隔</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>base中需要对比的表格，如果是多个表格以英文逗号分隔</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_showResult">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>1011</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>1011</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="button_SelectNewFile">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>new</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="Label_ShowNewFlie">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>x</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="button_SelectBaseFile">
       <property name="text">
        <string>base</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="Label_ShowBaseFlie">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>x</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="checkBox_newjudgeIn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>打钩表示只保留筛选内容对应的行，不打钩表示排除筛选内容对应的行</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_basejudgeIn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>360</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>打钩表示只保留筛选内容对应的行，不打钩表示排除筛选内容对应的行</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1092</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
